### PR TITLE
[homematic] correct SCA warnings - README.md

### PR DESCRIPTION
**Pull request by Rosi2143**
make changes to README.md

SCA Jenkins analysis
* https://ci.openhab.org/job/PR-openHAB2-Addons/Static_20Code_20Analysis_20Report/

.binding.homematic/README.md
tool 	priority 	line 	category 	rule 	message
checkstyle 	2 	46 	style 	MarkdownCheck 	The line before a Markdown list must be empty.
checkstyle 	2 	66 	style 	MarkdownCheck 	The line before a Markdown list must be empty.
checkstyle 	2 	92 	style 	MarkdownCheck 	The line before a Markdown list must be empty.
checkstyle 	2 	179 	style 	MarkdownCheck 	The line after a Markdown list must be empty.
checkstyle 	2 	269 	style 	MarkdownCheck 	The line before code formatting section must be empty.
checkstyle 	2 	463 	style 	MarkdownCheck 	The line before a Markdown list must be empty.

Also-by: Michael Roßner <Schrott.Micha@web.de>
Signed-off-by: Rosi2143 <Schrott.Micha@web.de>

### CODE REVIEW

File: bundles/org.openhab.binding.homematic/README.md [hunk 1]
```diff
@@ -177,8 +180,7 @@ If set to true, devices are automatically unpaired from the gateway when their c
 **Warning!** The option "factoryResetOnDeletion" also unpairs a device, so in order to avoid unpairing on deletion completely, both options need to be set to false! (default = false)
 
 -   **factoryResetOnDeletion**
-If set to true, devices are automatically factory reset when their corresponding things are removed.
-Due to the factory reset, the device will also be unpaired from the gateway, even if "unpairOnDeletion" is set to false! (default = false)
+If set to true, devices are automatically factory reset when their corresponding things are removed. Due to the factory reset, the device will also be unpaired from the gateway, even if "unpairOnDeletion" is set to false! (default = false)
```
Comment (martinvw) [inline bundles/org.openhab.binding.homematic/README.md]: Why did you join those sentences, AFAIK its prefered to have once line per sentence?

Comment (Rosi2143) [reply bundles/org.openhab.binding.homematic/README.md]: @martinvw I was considering it, but I gave the rule checker a higher prio than the "one line per sentence" guideline.
If you like, I can revert this part of the change again.

Comment (martinvw) [reply bundles/org.openhab.binding.homematic/README.md]: Yes please undo. @Confectrician is that a false positive of the checks which should be fixed?

Comment (Rosi2143) [reply bundles/org.openhab.binding.homematic/README.md]: I added the \n to line 181 again

**Pull request merged**